lib/logstash/inputs/syslog.rb

```diff
@@ -165,7 +184,7 @@ def tcp_listener(output_queue)
 
   public
   def teardown
-    @shutdown_requested = true
+    @shutdown_requested.update{|v| true}
```

Comment:
I found the second and the last one easier to read.
I have looked in our STYLE.md and we don't anything for that.

```ruby
@shutdown_requested.update{|v| true}
@shutdown_requested.update {|v| true}
@shutdown_requested.update { |v| true }
```
I could be picky and check for 80 characters line :boom: